src/classMembersNameRule.ts

```diff
@@ -266,12 +259,12 @@ class ClassMembersWalker extends Lint.ProgramAwareRuleWalker {
     }
 
     public visitGetAccessor(node: ts.GetAccessorDeclaration): void {
-        this.checkMethod(node, node.name, MemberKind.Getter);
+        this.checkMethod(node, node.name, MemberKind.Property);
         super.visitGetAccessor(node);
     }
 
     public visitSetAccessor(node: ts.SetAccessorDeclaration): void {
-        this.checkMethod(node, node.name, MemberKind.Setter);
+        this.checkMethod(node, node.name, MemberKind.Property);

```

Comment:
Treating `Getter` and `Setter` as `Property`?

Reply:
I don't see the point to have custom style for `Getter` and `Setter`. They behave like property.
```ts
interface Foo {
    name: string; // PropertySignature
}

class Bar implements Foo {
    // GetAccessorDeclaration
    public get name(): string {
          return "Bar";
    }
}
```